OJP/OJP_JourneySupport.xsd

```diff
@@ -516,9 +535,9 @@
 				</xs:element>
 				<xs:group ref="NumberOfResultsGroup" minOccurs="0"/>
 			</xs:choice>
-			<xs:element name="IgnoreRealtimeData" type="xs:boolean" default="false" minOccurs="0">
+			<xs:element name="UseRealtimeData" type="UseRealtimeDataEnumeration" default="full" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>The trip calculation should not use any realtime or incident data.</xs:documentation>
+					<xs:documentation>The way realtime data is used in the calculation of the trip (repleases IgnoreRealtimeData)</xs:documentation>
```

Comment:
`<xs:documentation>The way realtime data should be used in the calculation of the trip</xs:documentation>`

Regarding " (repleases IgnoreRealtimeData)": We did not add "historical" info about renamed/removed elements to the xsd yet, this rather belongs to the changes document. Even if this should be added, it should be "replaces" instead.

Reply:
changed